tests/assets/layers/cluster/layer.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Example of saving away a configuration, which can be imported from the bundled test library.